### DISCUSSION TITLE
How to take a screenshot?

### DISCUSSION THREAD

**Discussion by felixding** [trunkmaster/nextspace#402] [Q&A]
I was trying to report another issue about Login app, only to find I didn't know how to take screenshots on NEXTSPACE.

So far I tried with `PrtSc` key, Shutter and flameshot. None worked.

What's the correct way to do so?

**Comment by trunkmaster** [comment]
You can take screenshot with X11 utility. Take a look at this discussion https://askubuntu.com/questions/226829/how-to-take-screenshot-of-an-x11-based-gui-from-a-text-terminal-such-as-tty1.
